packages/nextjs/app/builders/0x726F186C449c58B07EfD0d1490B79F80843C2dB7/page.tsx

```diff
@@ -0,0 +1,92 @@
+import Link from "next/link";
+import type { NextPage } from "next";
+import { ArrowLeftIcon } from "@heroicons/react/24/outline";
+
+const BuilderPage: NextPage = () => {
+  return (
+    <>
+      <Link href="/builders" passHref>
+        <div className="flex items-center text-blue-500 hover:text-blue-700">
+          <ArrowLeftIcon className="h-5 w-5 mr-2" />
+          Back to Builders
+        </div>
+      </Link>
```

Comment:
This doesn't exist now, but it's ok... it'll soon :P

Reply:
🤦‍♂️🤦‍♂️🤦‍♂️